catalogue/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -95,7 +95,7 @@ type SeeMoreButtonType = {
 
 const SeeMoreButton = ({ text, link, totalResults }: SeeMoreButtonType) => (
   <MoreLink
-    name={`${text} (${formatNumber(totalResults)})`}
+    name={`${text} (${formatNumber(totalResults, true)})`}
```

Comment:
Can we make this a named parameter? A naked `true` or `false` isn't very readable.

Reply:
I wondered if we could just rely on the function description but yes let's make it readable at first glance!